src/rawchannel.cc

```diff
@@ -430,7 +506,7 @@ class RawChannel : public Nan::ObjectWrap
     RawChannel* hw = ObjectWrap::Unwrap<RawChannel>(info.Holder());
     CHECK_CONDITION(hw->IsValid(), "Channel not ready");
     const int loopback = 0;
-    setsockopt(hw->m_SocketFd, SOL_CAN_RAW, CAN_RAW_LOOPBACK, &loopback, sizeof(loopback));
+    setsockopt(hw->m_SocketFd, SOL_CAN_RAW, CAN_RAW_FILTER, NULL, 0);
```

Comment:
this change breaks loopback setting

Reply:
Code restaured, no need to modify this section (test done)